server/index.js

```diff
@@ -53,10 +56,10 @@ app.put('/api/description/:gameid', async (req,res) => {
   const { gameid } = req.params;
   const info = req.body;
 
-  if (gameid < 1 || gameid > 100) {
-    res.status(400).json({ error: 'Invalid game id' });
-    return;
-  }
+  // if (gameid < 1 || gameid > 100) {
```

Comment:
Maybe that 10000000 should be some kind of constant variable so it's clear what it represents.